app/src/main/java/com/xmartlabs/template/common/LoggedException.java

```diff
@@ -0,0 +1,7 @@
+package com.xmartlabs.template.common;
+
+/**
+ * Created by mirland on 13/04/2016.
+ */
+public class LoggedException extends Throwable {
```

Comment:
This name doesn't convince me. These exceptions are not logged, so it should be like `AlreadyLoggedException` or similar, doesn't it?